### PR TITLE
[DR-1675] Change get campaign response in adapter

### PR DESCRIPTION
**Pull request by mvillaseco**
# Overview
 Fix issue with return response from adapter

### CODE REVIEW

File: src/wwwroot/template-editor/adapterService.js [hunk 1]
```diff
@@ -396,9 +396,10 @@
           id: id,
           type: 'campaign',
           name: response.data.name,
+          html: response.data.html,
           attributes: response.data.mseditor ? response.data.mseditor.attributes : null,
-          innerHTML: response.data.mseditor ? response.data.mseditor.innerHTML : null,
-          children: response.data.mseditor ? response.data.mseditor.children : null
+          settings: response.data.mseditor ? response.data.mseditor.settings : null,
+          children: response.data.mseditor ? JSON.parse(response.data.mseditor.children) : null
```
Comment (andresmoschini) [inline src/wwwroot/template-editor/adapterService.js]: It should not be necessary.

Comment (cristiangiagante) [reply src/wwwroot/template-editor/adapterService.js]: I guess it depends how to children is returned from backend.

Comment (andresmoschini) [reply src/wwwroot/template-editor/adapterService.js]: Right, if it is necessary to parse a JSON, there is something wrong.

**Comment by andresmoschini** [comment]
Nice! this PR is really small now.

**Pull request merged**